index.html

```diff
@@ -5,24 +5,47 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
-  <script defer src="index.js"></script>
+  <script defer src="index.js"></script>            
   <title>Awesome Books</title>
 </head>
 <body>
-  <section>
-    <h1>Awesome Books</h1>
+  <nav class="nav flex space-btw">
+    <div class="logo">Awesome Books</div>
+    <ul class="nav-items flex">
+      <li class="items listBtn">List</li>
+      <li class="divider__line"></li>
+      <li class="items addBtn">Add New</li>
+      <li class="divider__line"></li>
+      <li class="items contactBtn">Contact</li>
+    </ul>
```

Comment:
Nice! You have three content sections - Books list, Add book form and Contact info ✔